### PR TITLE
adds script to test rhs time

### PR DESCRIPTION
**Pull request by yashu-seth**
- [x] There are no merge conflicts.
- [x] If there is a related issue, a reference to that issue is in the
  commit message.
- [x] Unit tests have been added for the new feature.
- [x] The PR passes tests both locally (run `nosetests`) and on Travis CI.
- [x] All public methods and classes have docstrings. (We use the [numpydoc
  format](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt).)
- [x] An explanation has been added to the online documentation. (`docs`
  directory)
- [x] The code follows PEP8 guidelines. (use a linter, e.g.
  [pylint](http://www.pylint.org), to check your code)
- [x] The new feature is documented in the [Release
  Notes](https://github.com/pydy/pydy#release-notes).
- [x] The code is backwards compatible. (All public methods/classes must
  follow deprecation cycles.)
- [x] All reviewer comments have been addressed.

### CODE REVIEW

**Comment by yashu-seth** [comment]
@moorepants Please have a look at this.

**Comment by yashu-seth** [comment]
For now I have added the script in tests.

File: pydy/tests/test_rhs_time.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    
+    time = timeit.repeat("rhs(x, 0.0, r[r_arg_type], p[p_arg_type])",
+                         "from __main__ import rhs,x,r,p,r_arg_type,p_arg_type", 
+                         number = itr)
+
+    print('For constants argument type - "{p_arg}" and '
+          'specifieds argument type - "{r_arg}" is '
+          .format(p_arg=p_arg_type, r_arg=r_arg_type))
+
+    print(sum(time)/3)
```
Comment (moorepants) [inline pydy/tests/test_rhs_time.py]: What is the divide by 3 for?

**Comment by moorepants** [comment]
This looks good but I think it should belong in the `bin` directory. Also please add a docstring at the top of the file that explains it's purpose and what it does.

**Comment by moorepants** [comment]
There seems to be some test failures wrt to importing.

File: pydy/tests/test_rhs_time.py (hunk 1 above)
Comment (yashu-seth) [inline pydy/tests/test_rhs_time.py]: ```timeit.repeat``` gives time for three repetitions in a list by default. So I took an average of the results. Should I change it?

**Comment by yashu-seth** [comment]
I'll move it to bin and the docstring.

File: pydy/tests/test_rhs_time.py (hunk 1 above)
Comment (moorepants) [inline pydy/tests/test_rhs_time.py]: Ok, got it. That's fine.

**Comment by moorepants** [comment]
The pep8 checks are failing on travis.

**Comment by yashu-seth** [comment]
@moorepants I cannot understand why travis is failing. The PR has no problem locally.

**Comment by yashu-seth** [comment]
okay i'll fix them.

**Comment by yashu-seth** [comment]
@moorepants I fixed the pep8 issues. But I don't understand why travis is still failing.

**Comment by moorepants** [comment]
`./bin/test_rhs_time.py:20:1: F401 'sp' imported but unused`

**Comment by yashu-seth** [comment]
got it, thanks

**Comment by moorepants** [comment]
One other note, please don't start the file name with "test" since it isn't a test. Maybe a better name would be `time_rhs.py`.

**Comment by yashu-seth** [comment]
@moorepants I have renamed the script as suggested.

**Comment by yashu-seth** [comment]
@moorepants For ```p_arg_type``` and ```r_arg_type``` both ```None``` we will again have a lot of permutations. Should I let the user decide what the input will be when ```constants_argument_type``` or ```specifieds_argument_type``` is ```None```. Same for the case when  ```constants_argument_type``` is ```dictionary```.
Or should I show the timing difference for all the permutations?

**Comment by moorepants** [comment]
Testing one permutation of the None arg type is enough. All it adds is a parsing function overhead to the specific declarations of that arg. It could even be ignored in this test as it is a constant wrt to the comparison you want to make.

**Comment by yashu-seth** [comment]
Yes I was also thinking the same. Okay let me make the changes.

**Pull request merged**